warehouse/query-core/src/main/java/datawave/query/jexl/lookups/LookupTermsFromRegex.java

```diff
@@ -159,8 +141,14 @@ public IndexLookupMap lookup(ShardQueryConfiguration config, ScannerFactory scan
                 
             }
             
-            for (Text field : fields) {
-                opts.fetchColumnFamily(field);
+            if (unfieldedLookup) {
```

Comment:
We are doing the same operation regardless of the flag. Can we remove the if/else here?

Reply:
oops....cut an past error.  good catch